runtime/doc/eval.txt

```diff
@@ -2476,6 +2477,20 @@ call({func}, {arglist} [, {dict}])			*call()* *E699*
 		{dict} is for functions with the "dict" attribute.  It will be
 		used to set the local variable "self". |Dictionary-function|
 
+capture({command})					*capture()*
+		Capture output of {command}.
+		If {command} is a |String|, it returns {command} output.
+		If {command} is a |List|, it returns concatenated outputs of
+		each items.
```

Comment:
“Items” is still plural.

---

By the way, I see `f_capture` is using msg_silent. If I am not mistaking, this is like `:silent` (without bang), but this fact is neither expressed in a help nor is tested. I think that

1. Here another paragraph is needed telling that “Commands are run as if they were prepended with |:silent| modifier.{two spaces}|:redir| and capture() work in parallel: capture() does not cancel |:redir|, |:redir| does not cancel capture().{two spaces}capture() inside capture() is allowed unlike nested |:redir|s, outer capture will not catch commands’ output of the inner one, but inner will neither cancel outer.”
2. Add test like

    ```lua
      it('silences command run inside', function()
          local screen = Screen.new()
          screen:attach()
          screen:set_default_attr_ignore( {{bold=true, foreground=255}} )
          feed(':let g:mes = capture("echo 42")<CR>')
          screen:expect([[
          ^                                                     |
          ~                                                    |
          ~                                                    |
          ~                                                    |
          ~                                                    |
          ~                                                    |
          ~                                                    |
          ~                                                    |
          ~                                                    |
          ~                                                    |
          ~                                                    |
          ~                                                    |
          [No Name]                                            |
                                                               |
        ]])
        eq('42', meths.get_var('msg'))
      end)
    ```

    (last line may actually be `:let …`, but in any case there should not be line containing “42”.)